### PR TITLE
Supporting getFormDefinition for UE

### PR DESCRIPTION
**Pull request by TalmizAhmed**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java [hunk 1]
```diff
@@ -362,4 +367,18 @@ public String getName() {
         return FormContainer.super.getName();
     }
 
+    @JsonIgnore
```
Comment (nit23uec) [inline bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java:370]: jsonignore should not be required in impl, I guess.

Comment (TalmizAhmed) [reply bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java:370]: I see it being used in other methods above of the same class, and so maybe it is?
cc: @rismehta

File: bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormContainer.java [hunk 2]
```diff
@@ -362,4 +362,14 @@ default void visit(Consumer<ComponentExporter> callback) throws Exception {}
...
         return null;
     }
+
+    /**
+     * Returns the form container's json.
+     * 
+     * @return A String that consists of the form container json
+     */
+    @JsonIgnore
+    default String getFormDefinition() {
```
Comment (rismehta) [inline bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormContainer.java:372]: This is already part of FormStructureParser and it is merged. Why do we have another PR for this ?

**Review by rismehta — CHANGES_REQUESTED** [review]
incorrect PR
